ModelBundle/Repository/NodeRepository.php

```diff
@@ -163,7 +163,7 @@ public function findPublishedInLastVersion($nodeId, $language, $siteId)
         $qa = $this->createAggregationQueryBuilderWithSiteIdAndLanguage($siteId, $language);
         $filter = array();
         if ($nodeId !== NodeInterface::TRANSVERSE_NODE_ID) {
-            $filter['status.published'] = true;
+            $filter['currentlyPublished'] = true;
```

Comment:
if you update the content the method maybe create new method and add deprecated on this method